<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select</class>
 <widget class="QWidget" name="select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>年份</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>至</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>至</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>481</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>(两位小数）</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>书号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>类别</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>书名</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>出版社</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>年份</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>作者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>价格</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>总藏书</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>库存</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>排序</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
